packages/host/app/components/card-catalog-modal.gts

```diff
@@ -54,30 +63,84 @@ export default class CardCatalogModal extends Component<Signature> {
                   data-test-create-new
                 >Create New</Button>
               {{/if}}
+              <div class='card-catalog__results-length'>{{if
+                  this.currentRequest.search.instances.length
+                  this.currentRequest.search.instances.length
+                  '0'
+                }}
+                results</div>
               <ul class='card-catalog' data-test-card-catalog>
                 {{#each this.currentRequest.search.instances as |card|}}
-                  <li data-test-card-catalog-item={{card.id}}>
+                  <li
+                    class={{cn
+                      'card-catalog-item'
+                      is-selected=(eq this.selectedCard.id card.id)
```

Comment:
Maybe modifier css class would be more appropriate? `card-catalog-item--selected`